web/pgadmin/browser/static/js/constants.js

```diff
@@ -46,6 +46,21 @@ export const BROWSER_PANELS = {
   WELCOME_PSQL_TOOL: 'id-welcome-psql'
 };
 
+export const MODAL_DIALOGS = {
+  PREFERENCES: 'id-preferences',
+  ABOUT: 'id-about',
+  DEBUGGER: 'id-debugger',
+  CONNECT_SERVER: 'id-connect-server',
+  MASTER_PASSWORD: 'id-master-password',
+  CHNAGE_PASSWORD: 'id-change-password',
+  CHANGE_OWNER: 'id-change-owner',
+  QUICK_SEARCH: 'id-quick-search',
+  USER_MANAGEMENT: 'id-user-management',
+  RESET_PREFERNCES: 'id-reset-preferences',
+  QT_CONFIRMATIONS: 'id-qt-close-confirmation-',
```

Comment:
hyphen at end?

Reply:
While testing I have used same db to open multiple pts, hence did not realise that those are independent. Removed transaction id.